app/tests/spec/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'sinon',
+  'underscore',
+  'views/mixins/modal-settings-panel-mixin',
+  'views/base',
+  'lib/metrics',
+  'stache!templates/test_template',
+  '../../../lib/helpers'
+], function (Chai, sinon, _, ModalSettingsPanelMixin, BaseView,
```

Comment:
We've been using lowcase `c` for `chai` elsewhere I believe. I like the C, but think we should do it universally.